### DISCUSSION TITLE
[Supply] Google Api Error: Server error - Internal error encountered.

### DISCUSSION THREAD

**Discussion by FlaShG** [fastlane/fastlane#18417] [Q&A] [answered]
Hello!

After running into an issue that is most likely #18403, we applied the proposed fix and downgraded `declarative-option`. I don't know whether this is related, but since then, we've been consistently getting
```Google Api Error: Server error - Internal error encountered.```
while running `Supply`. I've not been successful in getting any further information as to what exactly is going on. "Google is just having a bad day all day" is not really taking the top spot in my list of possible problems, but I've run out of ideas to fill that list anyway.

Does anyone have an idea what I could try to find out more, and to eventually get our pipeline running again?

Thank you!

P.S. Sorry if I missed obvious data I should have attached - I'm relatively new in working with fastlane 😅

**Comment by FlaShG** [comment, marked as answer]
Aaand I see that this has already come up in the issue thread (https://github.com/fastlane/fastlane/issues/18403#issuecomment-801256013) I'll look into this first...

Yep, turns out that opening the play console and telling google whether or not our app is a "news app" fixes the issue. Oof 😄

**Comment by bhattanmol101** [comment]
I am getting the same error my app has been posted everything has been connected and it worked once but after that, it just stopped working no help from google play support as well please help stuck here.

`fastlane supply init`

giving error

`Google Api Error: Server error - Internal error encountered.`


Everything seems to be in order but still not working please help.

Checking logs of my account I see this particular thing fails  `google.play.publishingapi.v3.EditsService.Insert`

**Reply by michaelknoch** [reply]
I'm facing a similar issue. Can you explain how to check the logs of the account? @bhattanmol101

**Reply by uchar** [reply]
same issue here , our pipeline was working fine for 2 years but suddenly today it's not working

**Reply by KrivtsovIvan** [reply]
This app uses an unsupported version of Google Play Payments. To publish it, update the Payments library to version 4 or later. - 
https://developer.android.com/google/play/billing/deprecation-faq.

**Reply by bhattanmol101** [reply]
> I'm facing a similar issue. Can you explain how to check the logs of the account? @bhattanmol101

That was in the google cloud console if you check the Google API calls to your account.
Don't remember where exactly

**Comment by felipevyg** [comment]
Don't use the BILLING permission on the manifest. Add the billing client dependency in your build.gradle file:
```
dependencies {
    implementation 'com.android.billingclient:billing:4.0.0'
}
```
